publications-2022.bib

```diff
@@ -574,6 +566,15 @@ @Article{2022:stein.comeau.ea:numerical
   url     = {https://www.sciencedirect.com/science/article/pii/S0040195122000701}
 }
 
+@InProceedings{2022:thiele.wick:space-time,
+  author  = {J. P. Thiele and T. Wick},
+  title   = {Space-time error control using a partition-of-unity dual-weighted residual method applied to low mach number combustion},
+  booktitle = {Proceedings of ICOSAHOM, 2020+1},
```

Comment:
Is this really the year, 2020+1?

If so, please merge yourself.

Reply:
I've checked their website: https://www.icosahom2020.org/proceedings

The conference was supposed to happen in 2020, but because of the pandemic they moved it to 2021.
![image](https://user-images.githubusercontent.com/18285973/190825727-6cbb5bc8-6574-444e-abfa-73cc5e9b692f.png)

So the conference actually happened in 2021. Let me move the entry to the 2021 year then.